src/django_ixctl/rest/urls/ixctl.py

```diff
@@ -1,8 +1,48 @@
 from django.urls import path, include
-
-import django_ixctl.rest.views.ixctl
+from django_ixctl.rest.views.ixctl import InternetExchange, Members, Routeserver
 import django_ixctl.rest.route.ixctl
 
+ix_list = InternetExchange.as_view({
```

Comment:
@egfrank i think if we switch the viewset class to `ModelViewset` those method to function assigments should happen automatically

Reply:
I think doing `viewset.as_view()` still requires rewriting the actions.
http://www.cdrf.co/3.1/rest_framework.viewsets/ModelViewSet.html#as_view

Reply:
Ah ok